modules/postgresql/variables.tf

```diff
@@ -269,3 +269,9 @@ variable "deletion_protection" {
   type        = bool
   default     = true
 }
+
+variable "read_replica_deletion_protection" {
+  description = "Used to block Terraform from deleting replica SQL Instances."
+  type        = bool
+  default     = true
```

Comment:
I'm wondering if this should be false by default? Deleting read replicas isn't as dangerous as deleting the primary.

Reply:
Updated, also removed `read_replica_deletion_protection = false` from the examples